packages/openapi-typescript/bin/cli.js

```diff
@@ -140,15 +136,24 @@ async function main() {
     console.info(HELP);
     process.exit(0);
   }
-  const packageJSON = JSON.parse(fs.readFileSync(new URL("../package.json", import.meta.url), "utf8"));
+  const packageJSON = (await import("../package.json", { assert: { type: 'json' }})).default;
```

Comment:
I really like import assertions! But let’s not use it here as it’s only stabilized in v20 and we still want to support v18 (and unofficially we support v16 for the next month until it’s EOL)